tests/test_customer_subscriptions.py

```diff
@@ -129,3 +130,15 @@ def test_update_customer_subscription(client, response):
     subscription = client.customer_subscriptions.with_parent_id(CUSTOMER_ID).update(SUBSCRIPTION_ID, data)
     assert isinstance(subscription, Subscription)
     assert subscription.id == SUBSCRIPTION_ID
+
+
+def test_list_subscription_payments(client, response):
```

Comment:
Please use the same test naming scheme as in other tests: `test_<current object>_get_related_<related data>`, see f.i. `test_customer_subscription_get_related_customer`. This would result in: `test_customer_subscription_get_related_payments`